pkg/ccl/backupccl/backup_planning.go

```diff
@@ -383,8 +383,17 @@ func GetRedactedBackupNode(
 		return nil, err
 	}
 
+	sanitizedIncstorage := make([]string, len(incrementalStorage))
+	for i, uri := range incrementalStorage {
+		var err error
+		sanitizedIncstorage[i], err = cloud.SanitizeExternalStorageURI(uri, nil /* extraParams */)
+		if err != nil {
+			return nil, err
+		}
+	}
+
 	resolvedOpts, err := resolveOptionsForBackupJobDescription(backup.Options, kmsURIs,
```

Comment:
aren't we sanitizing the passed in `incrementalStorage` inside this method:
https://github.com/dt/cockroach/blob/cee009fed948c7741ac973bf0c3c040f10799ded/pkg/ccl/backupccl/backup_planning.go#L338

Reply:
ah, yeah. this one might be superfluous then and we just needed the schedule one.